internal/k8s/service/resolver.go

```diff
@@ -165,22 +179,16 @@ type ResolvedReference struct {
 	Type      ResolvedReferenceType
 	Reference *BackendReference
 	Consul    *ConsulService
-	object    client.Object
```

Comment:
`object` was only here for the `Item()` func below which is now completely unused